encoders/lendianSteganography.py

```diff
@@ -0,0 +1,89 @@
+# Lendian stegonography system
```

Comment:
It does work on images, technically. Right now it can only encode text into random data that it generates.